src/index.html

```diff
@@ -11,7 +11,204 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
```

Comment:
name and id values should use camelCase, proper version would be `surname`. Fix it for all such values
```suggestion
            name="surname"
            id="surname"
```